### PR TITLE
up-to-date french translation

### PR DESCRIPTION
**Pull request by tcitworld**

### CODE REVIEW

File: app/src/main/res/values-fr/strings.xml [hunk 1]
```diff
@@ -126,4 +126,95 @@ GAULUPEAU Jonathan — 2013
...
+    <string name="connectionWizard_welcome_next">Suivant</string>
+    <string name="connectionWizard_welcome_prev">Ignorer</string>
+    <string name="connectionWizard_welcome_rerunNotice">Vous pouvez relancer l\'assistant de configuration depuis les paramètres de l\'application</string>
+    <string name="connectionWizard_welcome_text">Avant que vous puissiez utiliser l\'application wallabag, vous devez mettre en place la connexion. Cet assistant de connexion vous aidera à effectuer cette tâche.</string>
+    <string name="connectionWizard_welcome_titleMessage">Bienvenue sur wallabag !</string>
+    <string name="d_checkFeeds_errorMessage_noAccess">Aucun accès aux flux</string>
+    <string name="d_checkFeeds_errorMessage_notFound">Flux non trouvés</string>
+    <string name="d_checkFeeds_errorMessage_unknown">Erreur inconnue</string>
+    <string name="d_checkFeeds_errorMessage_unknownWithMessage">Erreur inconnue : %s</string>
+    <string name="d_checkFeeds_title_fail">Échec du test des flux </string>
```
Comment (nicosomb) [inline app/src/main/res/values-fr/strings.xml]: useless space at the end of the sentence :trollface:

File: app/src/main/res/values-fr/strings.xml [hunk 2]
```diff
@@ -126,4 +126,95 @@ GAULUPEAU Jonathan — 2013
...
+    <string name="d_configurationChanged_message">Il semble que certains paramètres aient été modifiés. Il est recommandé d\'effectuer un test de connexion.</string>
+    <string name="d_configurationChanged_title">Paramètres modifiés</string>
+    <string name="d_configurationIsQuestionable_message">Il y a eu une erreur de connexion récemment. Il est recommandé d\'effectuer un test de connexion.</string>
+    <string name="d_configurationIsQuestionable_title">Problème détecté</string>
+    <string name="d_getFeedsCredentials_title_fail">Impossible de configurer les flux</string>
+    <string name="d_getFeedsCredentials_title_message">Quelque chose s\'est mal passé. Vérifiez votre connexion internet et essayez à nouveau.</string>
+    <string name="d_testConnection_urlSuggestion_acceptButton">Utiliser l\'URL suggérée</string>
+    <string name="d_testConnection_urlSuggestion_cancelButton">Annuler</string>
+    <string name="d_testConnection_urlSuggestion_declineButton">Utiliser mon URL</string>
+    <string name="d_testConnection_urlSuggestion_message"> Il est recommandé de changer l\'URL à : "%s". Mais vous pouvez également utiliser celle tapée.</string>
```
Comment (nicosomb) [inline app/src/main/res/values-fr/strings.xml]: Useless space at the start of the line.

File: app/src/main/res/values-fr/strings.xml [hunk 3]
```diff
@@ -126,4 +126,95 @@ GAULUPEAU Jonathan — 2013
...
+    <string name="pref_desc_connection_autofill">Tester la connexion et remplir les informations de version de serveur, l\'ID utilisateur et le token</string>
+    <string name="pref_desc_misc_wipeDB">Supprime tous les articles de la base de données locale, et enlève également tous les changements et URL non synchronisés.</string>
+    <string name="pref_desc_runConnectionWizard">L\'assistant de connexion va vous aider à configurer les informations de connexion de base</string>
+    <string name="pref_name_autoDlNew_enabled">Télécharger les nouveaux articles</string>
+    <string name="pref_name_autoSync_enabled">Activer la synchronisation automatique</string>
+    <string name="pref_name_autoSync_interval">Intervalle de synchronisation automatique</string>
+    <string name="pref_name_autoSync_type">Type de synchronisation automatique</string>
+    <string name="pref_name_connection_advanced_clearCookies">Nettoyer les cookies</string>
+    <string name="pref_name_connection_autofill">Remplir automatiquement</string>
+    <string name="pref_name_runConnectionWizard">Lançer l\'assistant de connexion</string>
```
Comment (nicosomb) [inline app/src/main/res/values-fr/strings.xml]: `Lancer`.

File: app/src/main/res/values-fr/strings.xml [hunk 4]
```diff
@@ -126,4 +126,95 @@ GAULUPEAU Jonathan — 2013
...
+    <string name="pref_option_autoSync_interval_15m">15 minutes</string>
+    <string name="pref_option_autoSync_interval_1h">1 heure</string>
+    <string name="pref_option_autoSync_interval_24h">24 heures</string>
+    <string name="pref_option_autoSync_interval_30m">30 minutes</string>
+    <string name="pref_toast_connection_advanced_clearCookies">Cookies nettoyés</string>
+    <string name="settings_parametersAreOk">Les paramètres sont corrects</string>
+    <string name="settings_parametersAutofilled">Paramètres récupérés</string>
+    <string name="testConnection_errorMessage_authProblems">Problèmes d\'autorisation. Ceux-ci sont souvent causés par des redirections HTTP</string>
+    <string name="testConnection_errorMessage_httpAuth">Échec de l\'authentification HTTP</string>
+    <string name="testConnection_errorMessage_incorrectServerVersion">Version du serveur choisie inccorecte</string>
```
Comment (nicosomb) [inline app/src/main/res/values-fr/strings.xml]: `incorrecte`

File: app/src/main/res/values-fr/strings.xml [hunk 5]
```diff
@@ -126,4 +126,95 @@ GAULUPEAU Jonathan — 2013
     <string name="pref_name_misc_wipeDB">Purger la base de données</string>
     <string name="pref_name_misc_wipeDB_confirmTitle">Purger la base de données ?</string>
     <string name="pref_name_misc_wipeDB_confirmMessage">Êtes vous certain de vouloir purger la base de données ?</string>
+    <string name="checkFeeds_progressDialogMessage">Vérification des flux</string>
+    <string name="connectionWizard_providerSelection_next">Suivant</string>
+    <string name="connectionWizard_providerSelection_prev">Ignorer</string>
+    <string name="connectionWizard_providerSelection_provider_framabag_desc">Choisissez cette option si vous avez un compte sur https://framabag.org</string>
+    <string name="connectionWizard_providerSelection_provider_framabag_name">Framabag.org</string>
```
Comment (Strubbl) [inline app/src/main/res/values-fr/strings.xml]: when my PR is merged, this line is going to be unnecessary. please see 
https://github.com/wallabag/android-app/pull/297/files#diff-01dafb3fd0217441330103cfc8300094L223

File: app/src/main/res/values-fr/strings.xml [hunk 6]
```diff
@@ -126,4 +126,95 @@ GAULUPEAU Jonathan — 2013
...
+    <string name="connectionWizard_providerSelection_provider_general_desc">Choisissez cette option pour entrer les paramètres requis manuellement</string>
+    <string name="connectionWizard_providerSelection_provider_general_name">Autre</string>
+    <string name="connectionWizard_providerSelection_provider_v2wallabagorg_desc">Choisissez cette option si vous avez un compte sur https://v2.wallabag.org</string>
+    <string name="connectionWizard_providerSelection_provider_v2wallabagorg_name">v2.wallabag.org</string>
```
Comment (Strubbl) [inline app/src/main/res/values-fr/strings.xml]: when my PR is merged, this line is going to be unnecessary
https://github.com/wallabag/android-app/pull/297/files#diff-01dafb3fd0217441330103cfc8300094L221

File: app/src/main/res/values-fr/strings.xml [hunk 7]
```diff
@@ -126,4 +126,95 @@ GAULUPEAU Jonathan — 2013
...
+    <string name="connectionWizard_summary_next">Sauvegarder les paramètres</string>
+    <string name="connectionWizard_summary_prev">Retour</string>
+    <string name="connectionWizard_summary_text">Appuyez sur le bouton pour sauvegarder les paramètres et aller à l\'écran principal</string>
+    <string name="connectionWizard_summary_titleMessage">C\'est fini !</string>
+    <string name="connectionWizard_summary_toastMessage">La configuration est achevée</string>
+    <string name="connectionWizard_welcome_next">Suivant</string>
+    <string name="connectionWizard_welcome_prev">Ignorer</string>
+    <string name="connectionWizard_welcome_rerunNotice">Vous pouvez relancer l\'assistant de configuration depuis les paramètres de l\'application</string>
+    <string name="connectionWizard_welcome_text">Avant que vous puissiez utiliser l\'application wallabag, vous devez mettre en place la connexion. Cet assistant de connexion vous aidera à effectuer cette tâche.</string>
+    <string name="connectionWizard_welcome_titleMessage">Bienvenue sur wallabag !</string>
```
Comment (Strubbl) [inline app/src/main/res/values-fr/strings.xml]: useless space before exclamation point

File: app/src/main/res/values-fr/strings.xml [hunk 8]
```diff
@@ -126,4 +126,95 @@ GAULUPEAU Jonathan — 2013
...
+    <string name="d_checkFeeds_errorMessage_noAccess">Aucun accès aux flux</string>
+    <string name="d_checkFeeds_errorMessage_notFound">Flux non trouvés</string>
+    <string name="d_checkFeeds_errorMessage_unknown">Erreur inconnue</string>
+    <string name="d_checkFeeds_errorMessage_unknownWithMessage">Erreur inconnue : %s</string>
```
Comment (Strubbl) [inline app/src/main/res/values-fr/strings.xml]: useless space before colon

File: app/src/main/res/values-fr/strings.xml [hunk 9]
```diff
@@ -126,4 +126,95 @@ GAULUPEAU Jonathan — 2013
...
+    <string name="d_configurationChanged_title">Paramètres modifiés</string>
+    <string name="d_configurationIsQuestionable_message">Il y a eu une erreur de connexion récemment. Il est recommandé d\'effectuer un test de connexion.</string>
+    <string name="d_configurationIsQuestionable_title">Problème détecté</string>
+    <string name="d_getFeedsCredentials_title_fail">Impossible de configurer les flux</string>
+    <string name="d_getFeedsCredentials_title_message">Quelque chose s\'est mal passé. Vérifiez votre connexion internet et essayez à nouveau.</string>
+    <string name="d_testConnection_urlSuggestion_acceptButton">Utiliser l\'URL suggérée</string>
+    <string name="d_testConnection_urlSuggestion_cancelButton">Annuler</string>
+    <string name="d_testConnection_urlSuggestion_declineButton">Utiliser mon URL</string>
+    <string name="d_testConnection_urlSuggestion_message">Il est recommandé de changer l\'URL à : "%s". Mais vous pouvez également utiliser celle tapée.</string>
+    <string name="d_testConnection_urlSuggestion_message_mandatory">Il est recommandé de changer l\'URL à : "%s"</string>
```
Comment (Strubbl) [inline app/src/main/res/values-fr/strings.xml]: useless space before colon

File: app/src/main/res/values-fr/strings.xml [hunk 10]
```diff
@@ -126,4 +126,95 @@ GAULUPEAU Jonathan — 2013
...
+    <string name="d_configurationChanged_message">Il semble que certains paramètres aient été modifiés. Il est recommandé d\'effectuer un test de connexion.</string>
+    <string name="d_configurationChanged_title">Paramètres modifiés</string>
+    <string name="d_configurationIsQuestionable_message">Il y a eu une erreur de connexion récemment. Il est recommandé d\'effectuer un test de connexion.</string>
+    <string name="d_configurationIsQuestionable_title">Problème détecté</string>
+    <string name="d_getFeedsCredentials_title_fail">Impossible de configurer les flux</string>
+    <string name="d_getFeedsCredentials_title_message">Quelque chose s\'est mal passé. Vérifiez votre connexion internet et essayez à nouveau.</string>
+    <string name="d_testConnection_urlSuggestion_acceptButton">Utiliser l\'URL suggérée</string>
+    <string name="d_testConnection_urlSuggestion_cancelButton">Annuler</string>
+    <string name="d_testConnection_urlSuggestion_declineButton">Utiliser mon URL</string>
+    <string name="d_testConnection_urlSuggestion_message">Il est recommandé de changer l\'URL à : "%s". Mais vous pouvez également utiliser celle tapée.</string>
```
Comment (Strubbl) [inline app/src/main/res/values-fr/strings.xml]: useless space before colon

File: app/src/main/res/values-fr/strings.xml [hunk 11]
```diff
@@ -126,4 +126,95 @@ GAULUPEAU Jonathan — 2013
...
+    <string name="d_testConnection_urlSuggestion_message_mandatory">Il est recommandé de changer l\'URL à : "%s"</string>
+    <string name="d_testConnection_urlSuggestion_title">Changer l\'URL ?</string>
```
Comment (Strubbl) [inline app/src/main/res/values-fr/strings.xml]: useless space before question mark

File: app/src/main/res/values-fr/strings.xml [hunk 12]
```diff
@@ -126,4 +126,95 @@ GAULUPEAU Jonathan — 2013
...
+    <string name="getFeedsCredentials_progressDialogMessage">Configuration des flux...</string>
+    <string name="menu_syncLocalChanges">Synchroniser les changements locaux</string>
+    <string name="notification_expandedError">Erreur : %s</string>
```
Comment (Strubbl) [inline app/src/main/res/values-fr/strings.xml]: useless space before colon

File: app/src/main/res/values-fr/strings.xml [hunk 13]
```diff
@@ -126,4 +126,95 @@ GAULUPEAU Jonathan — 2013
...
+    <string name="d_configurationIsQuestionable_title">Problème détecté</string>
+    <string name="d_getFeedsCredentials_title_fail">Impossible de configurer les flux</string>
+    <string name="d_getFeedsCredentials_title_message">Quelque chose s\'est mal passé. Vérifiez votre connexion internet et essayez à nouveau.</string>
+    <string name="d_testConnection_urlSuggestion_acceptButton">Utiliser l\'URL suggérée</string>
+    <string name="d_testConnection_urlSuggestion_cancelButton">Annuler</string>
+    <string name="d_testConnection_urlSuggestion_declineButton">Utiliser mon URL</string>
+    <string name="d_testConnection_urlSuggestion_message">Il est recommandé de changer l\'URL à : "%s". Mais vous pouvez également utiliser celle tapée.</string>
+    <string name="d_testConnection_urlSuggestion_message_mandatory">Il est recommandé de changer l\'URL à : "%s"</string>
+    <string name="d_testConnection_urlSuggestion_title">Changer l\'URL ?</string>
+    <string name="getFeedsCredentials_progressDialogMessage">Configuration des flux...</string>
```
Comment (Strubbl) [inline app/src/main/res/values-fr/strings.xml]: use of three points instead of unicode char …

File: app/src/main/res/values-fr/strings.xml [hunk 14]
```diff
@@ -126,4 +126,95 @@ GAULUPEAU Jonathan — 2013
...
+    <string name="notification_unknownError">Erreur inconnue</string>
+    <string name="notification_updatingAllFeeds">Mise à jour de tous les flux</string>
+    <string name="notification_updatingFeeds">Mise à jour des flux</string>
+    <string name="notification_updatingSpecificFeed">Mise à jour du flux %s</string>
+    <string name="pref_categoryName_autoSync">Synchronisation automatique</string>
+    <string name="pref_categoryName_connection">Connexion</string>
+    <string name="pref_categoryName_connection_advanced">Paramètres de connexion avancés</string>
+    <string name="pref_categoryName_miscellaneous">Divers</string>
+    <string name="pref_categoryName_ui">Interface</string>
+    <string name="pref_desc_autoDlNew_enabled">Télécharger automatiquement les nouveaux articles directement après leur ajout par l\'application. Note : le téléchargement ne se fera pas si vous avez des changements locaux non synchronisés</string>
```
Comment (Strubbl) [inline app/src/main/res/values-fr/strings.xml]: useless space before colon

File: app/src/main/res/values-fr/strings.xml (hunk 5 above)
Comment (tcitworld) [inline app/src/main/res/values-fr/strings.xml]: Will clean them after.

File: app/src/main/res/values-fr/strings.xml (hunk 7 above)
Comment (tcitworld) [inline app/src/main/res/values-fr/strings.xml]: French typo is this way. :-)

File: app/src/main/res/values-fr/strings.xml (hunk 8 above)
Comment (tcitworld) [inline app/src/main/res/values-fr/strings.xml]: French typo is this way. :-)

File: app/src/main/res/values-fr/strings.xml (hunk 10 above)
Comment (tcitworld) [inline app/src/main/res/values-fr/strings.xml]: French typo is this way. :-)

File: app/src/main/res/values-fr/strings.xml (hunk 9 above)
Comment (tcitworld) [inline app/src/main/res/values-fr/strings.xml]: French typo is this way. :-)

File: app/src/main/res/values-fr/strings.xml (hunk 11 above)
Comment (tcitworld) [inline app/src/main/res/values-fr/strings.xml]: French typo is this way. :-)

File: app/src/main/res/values-fr/strings.xml (hunk 12 above)
Comment (tcitworld) [inline app/src/main/res/values-fr/strings.xml]: French typo is this way. :-)

File: app/src/main/res/values-fr/strings.xml (hunk 14 above)
Comment (tcitworld) [inline app/src/main/res/values-fr/strings.xml]: French typo is this way. :-)

**Pull request merged**